regression/basic-flatfield-estimation-plugin/tests/test_plugin.py

```diff
@@ -85,10 +94,9 @@ def test_estimate(gen_images: tuple[str, pathlib.Path]) -> None:
     ), f"{darkfield_out} not in {out_names}"  # noqa: S101
 
 
-def test_cli(gen_images: tuple[str, pathlib.Path]) -> None:
+def test_cli(gen_images: tuple[str, pathlib.Path, pathlib.Path]) -> None:
     """Test the CLI."""
-    pattern, inp_dir = gen_images  # type: ignore[misc]
-    out_dir = pathlib.Path(tempfile.mkdtemp(suffix="out_dir"))
+    pattern, inp_dir, out_dir = gen_images
 
     runner = typer.testing.CliRunner()
 
```

Comment:
test_estimate verify only if the darkfield and flatfield images are generated as an outputs but there should also be a test that verify also if the estimatations for darkfield and flatfield are correct???

Reply:
Please also add docstring for _load_img() function in utils.py

Reply:
> test_estimate verify only if the darkfield and flatfield images are generated as an outputs but there should also be a test that verify also if the estimatations for darkfield and flatfield are correct???

Those tests are done in the basic-py package and I only call their methods in the plugin.

Reply:
> Please also add docstring for _load_img() function in utils.py

Done